hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/CompactSplit.java

```diff
@@ -312,14 +316,20 @@ public void switchCompaction(boolean onOrOff) {
     if (onOrOff) {
       // re-create executor pool if compactions are disabled.
       if (!isCompactionsEnabled()) {
-        LOG.info("Re-Initializing compactions because user switched on compactions");
+        if (LOG.isInfoEnabled()) {
```

Comment:
I prefer we keep the old style.

Reply:
In fact in this class ,some logs using the {{LOG.isInfoEnabled}} or {{LOG.isDebugEnabled}}, and other logs using {{LOG.info}} or {{LOG.debug}} directly. If you do not like this, I would revert it.

Reply:
If we  want to align them, I prefer we open a new issue.